ui/raidboss/raidboss_config.ts

```diff
@@ -1453,20 +1464,23 @@ const userFileHandler: UserFileCallback = (
   if (!baseOptions.Triggers)
     return;
 
+  baseOptions.Triggers ??= [];
+  baseOptions.LoadedTriggers ??= [];
```

Comment:
I still think `LoadedTriggers` is an implementation detail and it doesn't make sense for it to be part of `Options`.  It's not something set by a user or the config.

If `filename` is part of `LooseTriggerSet`, then `Triggers` can just be updated in place as it used to be, right? That's what I was trying to suggest before.

Reply:
This is for trying not to modified object in place 🤔

Reply:
In general, I think "don't modify things in place" is a good approach, but usually that means there needs to be a clear data pipeline or data should be in temporary scoped variables.  I think passing things through `Options` that don't live in `Options` is strictly worse.

(Also, this is already being modified in place on line 1336.)

Since there's already a lot going on here, what do you think about an approach of: (1) keep modifying things in place / keep `Options` as-is without new members (2) follow up to change how triggers are loaded here and maybe return different things from this function, make it better?